app.py

```diff
@@ -1,50 +1,150 @@
 import os
+import time
+import json
 
+from redis import Redis
 from flask import (
     Flask, abort, Blueprint, flash, render_template, redirect, request,
-    url_for, flash, jsonify
+    url_for, flash, jsonify, send_from_directory, Markup, session
 )
 from loguru import logger
 from rq import Queue
+from rq.job import Job
 import rq_dashboard
-
 from worker import conn, redis_url
+from flask_socketio import SocketIO
 
 q = Queue(connection=conn)
 
-
-app = Flask(__name__)
+app = Flask(__name__, static_url_path='/static')
```

Comment:
abrichr 9 days ago Member
Why is this necessary? Can't we just leave it at static?